dev-libs/efl/efl-1.20.4.ebuild

```diff
@@ -0,0 +1,289 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+MY_P=${P/_/-}
+
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_SUB_PROJECT="core"
+	EGIT_URI_APPEND="${PN}"
+elif [[ *"${PV}" == *"_pre"* ]] ; then
+	MY_P=${P%%_*}
+	SRC_URI="https://download.enlightenment.org/pre-releases/${MY_P}.tar.xz"
+else
+	SRC_URI="https://download.enlightenment.org/rel/libs/${PN}/${MY_P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~x64-solaris ~x86-solaris"
+fi
+
+inherit enlightenment pax-utils
+
+DESCRIPTION="Enlightenment Foundation Libraries all-in-one package"
+
+LICENSE="BSD-2 GPL-2 LGPL-2.1 ZLIB"
+IUSE="+bmp debug drm +eet egl fbcon +fontconfig fribidi gif gles glib gnutls gstreamer harfbuzz +ico ibus jpeg2k libressl neon oldlua opengl ssl physics pixman +png postscript +ppm +psd pulseaudio raw scim sdl sound systemd tga tiff tslib unwind v4l valgrind wayland webp X xim xine xpm"
```

Comment:
[efl-1.20.4.patch.txt](https://github.com/gentoo/gentoo/files/1350477/efl-1.20.4.patch.txt)
The following USE flag should be added:

* `connman`, introduced upstream in commit https://git.enlightenment.org/core/efl.git/commit/?id=db4d4f8c875962e660aa0c5d7a6f6f957cc07899 to `Specify which network control backend to be used by Efl.Net.Session and Efl.Net.Control` (available since v1.19.0)
  * `RDEPEND` should be modified to include `connman? ( net-misc/connman )`
  * `E_ECONF` should be modified to include `--with-net-control=$(usex connman connman none)`

* `vnc`, introduced upstream in commit https://git.enlightenment.org/core/efl.git/commit/?id=bc6e8d2692084a808c0cf7012e7db4a9849601e7
  * `REQUIRED_USE` should be modified to include `vnc? ( X fbcon )`
  * `RDEPEND` should be modified to include `vnc? ( net-libs/libvncserver )`
  * `E_ECONF` should be modified to include `$(use_enable vnc vnc-server)`

* `examples` since configure option `--enable-always-build-examples` is now enable by default since  https://git.enlightenment.org/core/efl.git/commit/?id=b58629bbbef3d811700b016eb9fa97e2af71e1a1
  * `E_ECONF` should be modified to include `$(use_enable examples always-build-examples)`

* `gnutls` USE flag now need `>=net-libs/gnutls-3.3.6` as per https://git.enlightenment.org/core/efl.git/commit/?id=88faca55910dad33bb70d9758e3f2a1c865b7ba4

* `dev-libs/wayland` and `x11-libs/libxkbcommon` RDEPEND should be bumped to:
  * `>=dev-libs/wayland-1.11.0` as per https://git.enlightenment.org/core/efl.git/commit/?id=55750d41fad6055a549664ae92a34e636d7fb1f0
  * `>=x11-libs/libxkbcommon-0.6.0` https://git.enlightenment.org/core/efl.git/commit/?id=173fda5c11fc083a0b274e404f0005a52b4bf128

* `RDEPEND` need to be modified to add `>=media-libs/freetype-2.5.0.1` since it is required as per https://git.enlightenment.org/core/efl.git/commit/?id=4029b99bb42079aea35642197d77e66c89ca10b5

* `RDEPEND` need to be modified to remove `x11-libs/libdrm` as per https://git.enlightenment.org/core/efl.git/commit/?id=fc81a998c871513bb2cd794b26ff307a0c153c7c

Full patch is attached.